crates/rooch-rpc-server/src/server/btc_server.rs

```diff
@@ -66,9 +66,14 @@ impl BtcAPIServer for BtcServer {
 
         let global_state_filter =
             UTXOFilterView::into_global_state_filter(filter, resolve_address)?;
+        let limit_value = limit_of
```

Comment:
Line 48 checks the limit to ensure that the value of limit is between [DEFAULT_RESULT_LIMIT_USIZE, MAX_RESULT_LIMIT_USIZE]. Does the overflow case not have this line of check?
```
  let limit_of = min(
             limit.map(Into::into).unwrap_or(DEFAULT_RESULT_LIMIT_USIZE),
             MAX_RESULT_LIMIT_USIZE,
         );
```

Reply:
Yes, this is not a security issue, that code has been removed.

Reply:
Maybe I misunderstood, will those code cause security issue?

```rust
let limit_of = min(
             limit.map(Into::into).unwrap_or(DEFAULT_RESULT_LIMIT_USIZE),
             MAX_RESULT_LIMIT_USIZE,
         );
````

Reply:
It will not. Take a look at the case in the audit report to see if the limit is not processed in that case.